runner/src/config.rs

```diff
@@ -25,6 +25,7 @@ pub struct Config {
     pub vcpu: VCpu,
     #[serde(default)]
     pub platform: Platform,
+    pub benchmark: Benchmark,
```

Comment:
You should add `#[serde(default)]` on the benchmark field, that way the benchmark table is not required in the toml config, which makes it easier to create new configs.